libs/gretty/src/main/groovy/org/akhikhl/gretty/StartBaseTask.groovy

```diff
@@ -54,10 +52,6 @@ abstract class StartBaseTask extends DefaultTask {
   void action() {
     LauncherConfig config = getLauncherConfig()
     Launcher launcher = new DefaultLauncher(project, config)
-    if(config.serverConfig.liveReloadEnabled) {
-      launcher.optionalLiveReloadServer = new OptionalLiveReloadServer(new LiveReloadServer())
-    }
-    launcher.scannerManager = createScannerManager(config, launcher.optionalLiveReloadServer)
```

Comment:
This is removing too much, is it?
This patch would kill the only usage of `JDKScannerManager` and `JettyScannerManager`, and probably remove any redeploy-on-file-change functionality of Gretty, right? (I did not test it).
I suppose we would need to leave the call to `createScannerManager` in, and just remove the `OptionalLiveReloadServer` bits.

Reply:
You're right! I did what you suggested, please take a look. :)